app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# :nodoc:
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by(params[:id])
+    @images = @user.images.order(created_at: :desc)
```

Comment:
let's add an index method that will show all users, we will need to see other users in order to make friends after all

Reply:
I was saving that method for the friendship model, I add it up